pom.xml

```
@@ -153,15 +57,15 @@
 			<plugin>
 				<groupId>com.totalcross</groupId>
 				<artifactId>totalcross-maven-plugin</artifactId>
-				<version>1.1</version>
+				<version>1.1.7-20200803.213852-3</version>
```

```suggestion
				<version>1.2.0</version>
```
Use version 1.2.0 as it's the newest one, try to always stay up to date on totalcross-maven-plugin to avoid possible problems that are already solved.